db/migrate/20210203130641_backfill_hesa_sex_and_ethnicity_codes.rb

```diff
@@ -0,0 +1,10 @@
+class BackfillHesaSexAndEthnicityCodes < ActiveRecord::Migration[6.0]
+  def change
+    not_nil = ApplicationForm.all.where.not(equality_and_diversity: nil)
+    not_nil.each do |application_form|
+      application_form.equality_and_diversity['hesa_sex'] = application_form.equality_and_diversity['hesa_sex'].to_s
+      application_form.equality_and_diversity['hesa_ethnicity'] = application_form.equality_and_diversity['hesa_ethnicity'].to_s
+      application_form.save
```

Comment:
Just noticed that using `save` will trigger callbacks on the application form, which will in turn touch the application choices' `updated_at`, which will in turn cause them to appear as "changed" in the API, which we don't want.

We can avoid this by using `update_column` — suggest pulling out the whole `equality_and_diversity` hash, updating it, and then writing back? eg (untested)

```
equality_and_diversity = application_form.equality_and_diversity
equality_and_diversity['hesa_sex'] = equality_and_diversity['hesa_sex'].to_s
equality_and_diversity['hesa_ethnicity'] = equality_and_diversity['hesa_ethnicity'].to_s

application_form.update_column(:equality_and_diversity, equality_and_diversity)
```